### PR TITLE
Long convertor javadoc

### PR DESCRIPTION
**Pull request by peter-lawrey**

### CODE REVIEW

File: src/main/java/net/openhft/chronicle/wire/VanillaMethodReader.java [hunk 1]
```diff
@@ -414,7 +413,7 @@ else if (o2 != null)
 
     @NotNull
     protected MethodWireKey createWireKey(@NotNull Method m, String name) {
-        MethodId annotation = Annotations.getAnnotation(m, MethodId.class);
+        MethodId annotation = Jvm.findAnnotation(m, MethodId.class);
```
Comment (JerryShea) [inline src/main/java/net/openhft/chronicle/wire/VanillaMethodReader.java:416]: none of these annotation changes are related to javadoc. Are they meant to be here?

Comment (peter-lawrey) [reply src/main/java/net/openhft/chronicle/wire/VanillaMethodReader.java:416]: I will separate them into a Java21 improvement PR.

File: pom.xml [hunk 2]
```diff
@@ -96,6 +96,12 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
```
Comment (RobAustin) [inline pom.xml:99]: This is not a javadoc change !

**Review by RobAustin — APPROVED** [review]

**Comment by RobAustin** [comment]
It is failing on ARM - 
image

**Pull request merged**